edenai_apis/apis/astria/astria_api.py

```diff
@@ -26,30 +31,81 @@ def __init__(self, api_keys: Dict = {}) -> None:
         self.headers = {"authorization": f"Bearer {self.api_key}"}
 
     def image__generation_fine_tuning__create_project_async__launch_job(
-        self,
-        name: str,
-        description: str,
-        files: List[str],
-        files_url: List[str] = [],
-        base_project_id: Optional[int] = None,
+            self,
+            title: str,
+            class_name: str,
+            files: List[str] = [],
+            files_url: List[str] = [],
+            base_tune_id: Optional[int] = None,
     ) -> AsyncLaunchJobResponseType:
-        raise NotImplementedError
+        data = {
+            "tune[title]": title,
+            "tune[name]": class_name,
+            "tune[base_tune_id]": base_tune_id,
+            # "tune[callback]": 'https://optional-callback-url.com/to-your-service-when-ready?prompt_id=1'
+        }
+        for image in files:
+            image_data = load_image(image)  # Assuming image is a file path
+            files.append(("tune[images][]", image_data))
```

Comment:
Append to files which create a problem because files list already exist

Reply:
indeed, and that's a good thing :)

Reply:
maybe i'm missing something... but that should be good